src/Microsoft.AspNetCore.Buffering/baseline.netcore.json

```diff
@@ -0,0 +1,66 @@
+{
+  "AssemblyIdentity": "Microsoft.AspNetCore.Buffering, Version=0.4.1.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```

Comment:
I'll remove it from this PR, but please add `<EnableApiCheck>false</EnableApiCheck>` to it until it's a shipping product to avoid this happening next time.

Reply:
Can you do that in this PR?